quarkchain/cluster/tests/test_shard_state.py

```diff
@@ -3190,64 +3196,59 @@ def tx_gen(nonce, value, data: str):
+        set_auction_params = lambda: tx_gen(
+            "3c69e3d2" + parsed_hex(20) + parsed_hex(5) + parsed_hex(3600 * 24 * 7)
+        )
+        resume_auction = lambda: tx_gen("32353fbd")
+        bid_new_token = lambda v: tx_gen(
+            "6aecd9d7"
+            + parsed_hex(token_id)
+            + parsed_hex(25 * 10 ** 18)
+            + parsed_hex(0),
             v,
-            "3c69e3d2000000000000000000000000000000000000000000000000000000000000001400000000000000000000000000000000000000000000000000000000000000050000000000000000000000000000000000000000000000000000000000093a80",
         )
-        resume_auction = lambda n, v: tx_gen(n, v, "32353fbd")
-
-        # set auction params as (9999999, 25 * 10 ^ 18, 0)
-        bid_new_token = lambda n, v: tx_gen(
-            n,
-            v,
-            "6aecd9d7000000000000000000000000000000000000000000000000000000000098967f0000000000000000000000000000000000000000000000015af1d78b58c400000000000000000000000000000000000000000000000000000000000000000000",
-        )
-        end_auction = lambda n, v: tx_gen(n, v, "fe67a54b")
-        mint_new_token = lambda n, v: tx_gen(
-            n,
-            v,
-            "0f2dc31a000000000000000000000000000000000000000000000000000000000098967f00000000000000000000000000000000000000000000000000000000000003e8",
-        )
-        get_native_token_info = lambda n, v: tx_gen(
-            n,
-            v,
-            "9ea41be7000000000000000000000000000000000000000000000000000000000098967f",
+        end_auction = lambda: tx_gen("fe67a54b")
+        mint_new_token = lambda: tx_gen(
+            "0f2dc31a" + parsed_hex(token_id) + parsed_hex(amount)
         )
+        get_native_token_info = lambda: tx_gen("9ea41be7" + parsed_hex(token_id))
 
-        tx0 = set_auction_params(0, 0)
+        tx0 = set_auction_params()
         success, _ = apply_transaction(evm_state, tx0, bytes(32))
         self.assertTrue(success)
 
-        tx1 = resume_auction(1, 0)
+        tx1 = resume_auction()
         success, _ = apply_transaction(evm_state, tx1, bytes(32))
         self.assertTrue(success)
 
-        tx2 = bid_new_token(2, 26 * 10 ** 18)
+        tx2 = bid_new_token(26 * 10 ** 18)
         success, _ = apply_transaction(evm_state, tx2, bytes(32))
         self.assertTrue(success)
 
         # 7 days passed, this round of auction ends
         evm_state.timestamp += 3600 * 24 * 7
-        tx3 = end_auction(3, 0)
+        tx3 = end_auction()
         success, _ = apply_transaction(evm_state, tx3, bytes(32))
         self.assertTrue(success)
 
-        tx4 = get_native_token_info(4, 0)
+        tx4 = get_native_token_info()
         success, output = apply_transaction(evm_state, tx4, bytes(32))
         self.assertTrue(success)
-        self.assertEqual(int.from_bytes(output[:32], byteorder="big"), 604801)
+        self.assertNotEqual(int.from_bytes(output[:32], byteorder="big"), 0)
```

Comment:
the value should equal to the owner, which is `acc1.recipient`?

Reply:
It's the created time of queried token.